docs/manual/scala/guide/build/ScalaComponents.md

```diff
@@ -0,0 +1,96 @@
+# Scala Components
+
+The Lagom scaladsl requires building an Application that mixes in all the `Components` that will enable the required features for your code. This means each `Application` will need a specific set of cake slices. Not only that but `Dev` mode and `Prod` mode need adjusting since the environments where the Service is run vary.
```

Comment:
Yes, I think this page would be a better place for it. We also have to consider the order that people will see things in, if they read through the docs linearly.